sql/pgwire/types.go

```diff
@@ -64,7 +65,7 @@ func typeForDatum(d driver.Datum) pgType {
 		return pgType{oid.T_bool, 1, formatText}
 
 	case *driver.Datum_IntVal:
-		return pgType{oid.T_int8, 8, formatBinary}
+		return pgType{oid.T_int8, 8, formatText}
```

Comment:
The `pgType.preferredFormat` field is now `formatText` in all cases. Unless there are more uses for this field coming in future PRs it can just be removed.